subgraphs/compound-forks/protocols/bastion-protocol/README.md

```diff
@@ -1,6 +1,8 @@
 # Bastion Protocol Subgraph
 
-## Calculation Methodology v1.0.0
+## Calculation Methodology v1.0.1
```

Comment:
One of the downsides of this approach with adding the other realms is that the position data is expected to be pooled across the whole protocol, but now a position in a realm is siloed to that realm.

What I am referring to here is that compound positions are pooled together to get a liquidity in the protocol, where this is now siloed to each realm.

Reply:
Yes, that's true. At position level, it is hard to see which positions are pooled, which are isolated. An alternative is to have a separate subgraph for each realm - not sure that's much better (a lot duplicated code).

Reply:
Yep, we considered both options. I agree that would be worse as it seems strange to have 4 bastion subgraphs, I am just asking that we make note of the limitation to remain as transparent as possible.